lib/android-helpers.js

```diff
@@ -532,18 +532,34 @@ helpers.pushSettingsApp = async function pushSettingsApp (adb, throwError = fals
   // if the app is not launched prior to start the session on android 7+
   // see https://github.com/appium/appium/issues/8957
   try {
-    await adb.startApp({
-      pkg: SETTINGS_HELPER_PKG_ID,
-      activity: SETTINGS_HELPER_MAIN_ACTIVITY,
-      action: 'android.intent.action.MAIN',
-      category: 'android.intent.category.LAUNCHER',
-      flags: '0x10200000',
-      stopApp: false,
-    });
-  } catch (err) {
-    logger.warn(`Failed to launch settings app: ${err.message}`);
-    if (throwError) {
-      throw err;
+    const cmd = [
```

Comment:
interesting...
I tried this command on Android 4.4 (emulator), too. it worked as well as android 9 emulator.

```
WARNING: linker: libdvm.so has text relocations. This is wasting memory and is a security risk. Please fix.
Events injected: 1
## Network stats: elapsed time=80ms (0ms mobile, 0ms wifi, 80ms not connected)
```
the above status code is `0`..